pages/appPage/login_page.ts

```diff
@@ -17,18 +17,19 @@ export class AppLoginPage {
         await this.page.waitForLoadState('domcontentloaded')
         const login = await api.loginPage.login(`${url}:3000/login`)
         const user = await api.loginPage.addEmail(`${url}:3000/login`, login.token, apiDataSet.deviceUUID)
+        await this.page.reload()
         await this.page.evaluate(`window.localStorage.setItem('token', "${user.userToken}")`)
         await this.page.evaluate(`window.localStorage.setItem('isAuthorized', "true")`)
         await this.page.evaluate(`window.localStorage.setItem('streamData', 'null')`)
         const date = new Date().toISOString()
         await this.page.evaluate(`window.localStorage.setItem('lastGiftListRequestDate', '${date}')`)
+        await this.page.evaluate(`window.localStorage.setItem('profile', '${JSON.stringify(user.profile)}')`);
+        await this.page.reload()
         const userToken = user.userToken
         const id = user.id
         const name = user.name
         const humanReadableId = user.humanReadableId
         await api.slackPage.addCoins(humanReadableId)
-        await this.page.goto(`${apiUrl.qaUiUrl}/edit`)
-        await this.page.waitForTimeout(500)
```

Comment:
@YulianH instead of waitForTimeout you can use `await this.page.waitForLoadState('networkidle')`